clients/admin-ui/src/features/common/nav/v2/nav-config.ts

```diff
@@ -57,34 +57,31 @@ export const NAV_CONFIG: NavConfigGroup[] = [
         title: "View map",
         path: "/datamap",
         requiresPlus: true,
-        scopes: [ScopeRegistry.DATAMAP_READ],
+        scopes: [ScopeRegistryEnum.DATAMAP_READ],
       },
       {
         title: "View systems",
         path: "/system",
-        scopes: [ScopeRegistry.CLI_OBJECTS_READ],
+        scopes: [ScopeRegistryEnum.CLI_OBJECTS_READ],
       },
       {
         title: "Add systems",
         path: "/add-systems",
         scopes: [
-          ScopeRegistry.CLI_OBJECTS_CREATE,
-          ScopeRegistry.CLI_OBJECTS_UPDATE,
+          ScopeRegistryEnum.SYSTEM_CREATE,
+          ScopeRegistryEnum.SYSTEM_UPDATE,
         ],
       },
       {
         title: "Manage datasets",
         path: "/dataset",
-        scopes: [
-          ScopeRegistry.CLI_OBJECTS_CREATE,
-          ScopeRegistry.CLI_OBJECTS_UPDATE,
-        ],
+        scopes: [ScopeRegistryEnum.DATASET_CREATE_OR_UPDATE],
```

Comment:
I think the current scopes on the /dataset endpoints are CLI_OBJECTS_CREATE/UPDATE - the systems work will change so it has a scope of CTL_DATASET CREATE/UPDATE, but I don't think this one is right ^